README.md

```diff
@@ -20,13 +20,9 @@ If you are unfamiliar with these concepts you can catch up with:
 
     require 'sequent'
 
-    Sequent::Core::EventStore.configure do |config|
-      config.event_handlers = [MyEventHandler.new]
-    end
-
-    Sequent::Core::CommandService.configure do |config|
-      config.event_store = Sequent::Core::EventStore.instance
-      config.command_handler_classes = [MyCommandHandler]
+    Sequent.configure do |config|
+      config.event_handlers = [MyEventHandler]
```

Comment:
I can't find that anywhere in the code... See not below. Also in our current implementation we have some `EventHandler`s with collaborators. How would that work in this situation.
I do agree that both `CommandHandler`s and `EventHandler`s should be registered the same way. So either both as instances (my preferences) or both both only the classes. What do you think?